ckan/lib/search/__init__.py

```diff
@@ -1,318 +1,23 @@
-import logging
-import sys
-import cgitb
-import warnings
-import xml.dom.minidom
-import urllib2
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from ckan import plugins
 
-from pylons import config
-from paste.deploy.converters import asbool
 
-import ckan.model as model
-import ckan.plugins as p
-import ckan.logic as logic
+def index(package):
+    for plugin in plugins.PluginImplementations(plugins.IIndexer):
+        return plugin.index(package)
```

Comment:
Simply the pattern I copied from the existing code. If there's a better way I'm happily open to it.